internal/models/models.go

```diff
@@ -52,10 +52,17 @@ func (p Publisher) AfterSave(tx *gorm.DB) error {
 	return nil
 }
 
+func (CodeHosting) TableName() string {
+	return "publishers_code_hosting"
+}
+
```

Comment:
I think this is not needed

```suggestion
```

Reply:
Unfortunately, yes. Otherwise the table name assigned is code_hosting